java/display-grid/src/main/java/com/esri/arcgisruntime/sample/displaygrid/MainActivity.java

```diff
@@ -0,0 +1,241 @@
+  private Spinner mGridSpinner;
+  private Spinner mColorsSpinner;
+  private Button mMenuButton;
+  private Spinner mLabelColorSpinner;
+  private int mLineColor;
+  private int mLabelColor;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+
+    // inflate views from activity_main
+    mMapView = findViewById(R.id.mapView);
+    mMenuButton = findViewById(R.id.menu_button);
+
+    // set up a popup menu to manage grid settings
+    final AlertDialog.Builder builder = new AlertDialog.Builder(MainActivity.this);
+    final View view = getLayoutInflater().inflate(R.layout.popup_menu, null);
+    builder.setView(view);
+    final AlertDialog dialog = builder.create();
+
+    // inflate views from popup_menu
+    mGridSpinner = view.findViewById(R.id.layer_spinner);
+    mLabelsCheckBox = view.findViewById(R.id.labels_checkBox);
+    mColorsSpinner = view.findViewById(R.id.line_color_spinner);
+    mLabelColorSpinner = view.findViewById(R.id.label_color_spinner);
+
+    // create drop-down list of different grids
+    ArrayAdapter<String> adapter = new ArrayAdapter<>(MainActivity.this, android.R.layout.simple_spinner_item,
+        getResources().getStringArray(R.array.layers_array));
+    adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+    mGridSpinner.setAdapter(adapter);
+
+    // create drop-down list of different colors
+    ArrayAdapter<String> colorAdapter = new ArrayAdapter<>(MainActivity.this, android.R.layout.simple_spinner_item,
+        getResources().getStringArray(R.array.colors_array));
+    colorAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+    mColorsSpinner.setAdapter(colorAdapter);
+
+    // create drop-down list of different label colors
+    ArrayAdapter<String> labelColorAdapter = new ArrayAdapter<>(MainActivity.this, android.R.layout.simple_spinner_item,
+        getResources().getStringArray(R.array.colors_array));
+    labelColorAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+    mLabelColorSpinner.setAdapter(labelColorAdapter);
+
+    // create a map with imagery basemap
+    ArcGISMap map = new ArcGISMap(Basemap.createImagery());
+    // set viewpoint
+    final Point center = new Point(-7702852.905619, 6217972.345771, 23227, SpatialReference.create(3857));
+    map.setInitialViewpoint(new Viewpoint(center, 23227));
+    // set the map to be displayed in this view
+    mMapView.setMap(map);
+
+    // set defaults on grid
+    mMapView.setGrid(new LatitudeLongitudeGrid());
+    mLabelsCheckBox.setChecked(true);
+
+    // change different grids on the Map View
+    mGridSpinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+      @Override public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+        // set the grid type
+        switch (position) {
```

Comment:
this and other switches should all have a default branch. For example:

https://github.com/Esri/arcgis-runtime-samples-android/blob/426598127cf15b9a809ed714a10ac5675b0f746f/java/format-coordinates/src/main/java/com/esri/arcgisruntime/sample/formatcoordinates/MainActivity.java#L147